src/clr-angular/timeline/timeline-step-title.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'clr-timeline-step-title',
+  template: `
+        <ng-content></ng-content>
+    `,
+  host: { '[class.clr-timeline-step-title]': 'true', '[attr.aria-hidden]': 'true' },
```

Comment:
nit: would be good to add a comment when you squash about why this is hidden, it seems strange out of context.

Reply:
Added a note.